src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -782,19 +782,19 @@ private long newestVersion(DHTKey key) {
                             // they have all been gathered.
                         } else {
                             if (debugWaitFor) {
-                                System.out.printf("No waiter found for %s\n", pendingWaitFor.getOpUUID());
```

Comment:
debugWaitFor is off by default. Make Log.warningAsyncf so that the debugWaitFor flag has an effect